### PR TITLE
updating web pages-bootstrap- With Sign up option

### PR DESCRIPTION
**Pull request by erezdav22**
Upgraded the way the website looks by using Bootstrap.
Navigation in the site is as followed:
home page-> sign up/sign in -> personal area -> available jobs.

On the "available jobs" page we have at this point just a table that shows how the search results are going to be presented to the user. We will work on connecting it to the DB and presenting real search results in the table in the next sew days.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -5,12 +5,11 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
 
 [packages]
```
Comment (Yarboa) [inline Pipfile:10]: You have it in [dev-packages]
I think django is missing here, did you check if your server is running after vagrant up?

File: ultimatejobweb/boostraped_website_draft/sign_up.html [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (Yarboa) [inline ultimatejobweb/boostraped_website_draft/sign_up.html:3]: Why not to update the existing sign_up with this one?

File: ultimatejobweb/boostraped_website_draft/sign_in.html [hunk 3]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (Yarboa) [inline ultimatejobweb/boostraped_website_draft/sign_in.html:3]: Why not adding login page? you will have 
Home, Signup, Login, it is a progress

Comment (erezdav22) [reply ultimatejobweb/boostraped_website_draft/sign_in.html:3]: Hey,
just updated all the changes.
There is a login page, i just named it "sign in".
Right now the "sign up" option works and saves the details of the new user in the users DB using django form.
once a user signes up- he is redirected to his personal area, and from there he can go and see his job offers.

**Comment by Yarboa** [comment]
Nice work

**Pull request merged**